hugegraph-loader/assembly/static/bin/hugegraph-spark-loader.sh

```diff
@@ -25,7 +25,7 @@ get_params "$@"
 echo "engine_params: $ENGINE_PARAMS"
 echo "hugegraph_params: $HUGEGRAPH_PARAMS"
 
-ASSEMBLY_JAR_NAME=$(find "${LIB_DIR}" -name 'hugegraph-loader*.jar')
+ASSEMBLY_JAR_NAME=$(find "${LIB_DIR}" -name '*hugegraph-loader-incubating*.jar')
```

Comment:
```suggestion
ASSEMBLY_JAR_NAME=$(find "${LIB_DIR}" -name '*hugegraph-loader*.jar')
```

**test it** after we modify it (`incubating` word may remove in the future)

Reply:
ok

Reply:
> ok

seems also revert the `incubating` that we should remove? (Or we can't remove it after testing?)

Reply:
I will remove it.